apps/services/user-notification/src/app/modules/notifications/notifications.module.ts

```diff
@@ -74,6 +83,26 @@ import { NotificationsService } from './notifications.service'
           }),
         ),
     },
+    {
+      provide: userProfile.V2UsersApi,
```

Comment:
Does the notification module need both v1 and v2 of user profile?

Reply:
no - removing